stylus/cozy-ui/build.styl

```diff
@@ -361,6 +361,31 @@
 .c-textarea
     @extend $textarea
 
+/*
+ Select
+
+ Markup:
+ <form>
+    <div>
+        <select class="c-select {{modifier_class}}">
+            <option value="" disabled selected>Choose your option</option>
+            <option value="1">Option 1</option>
+            <option value="2">Option 2</option>
+            <option value="3">Option 3</option>
+            <option value="4">Option 4</option>
+        </select>
+    </div>
+ </form>
+
+ .is-error - Error variant
```

Comment:
Shouldn't this be a modifier class like .c-select--error ?

Reply:
Well I used the *SuitCSS* approach on this and prefered its [naming convention](https://github.com/suitcss/suit/blob/master/doc/naming-conventions.md), I used it a lot before but I may have forgot to update the Doc, sorry 😕. 
It's a lot like BEM actually but the main difference is it's seperating variant classes from state classes. 
I used this mainly for error and active states so far.

Reply:
That's the old style (keeping it for retrocompatibility) everything below [this line](https://github.com/GoOz/cozy-ui/blob/aa81512afe6c2752e7418ca646a07df1c0ce79cf/stylus/components/forms.styl#L229) is new 
I'm still refactoring bit by bit but the old style won't stay.